motoman_sda10f_support/test/sda10f_test.xacro

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" ?>
+
+<robot name="motoman_sda10f_test" xmlns:xacro="http://ros.org/wiki/xacro">
+	<xacro:include filename="$(find motoman_sda10f_support)/urdf/sda10f_macro.xacro" />
+	<xacro:motoman_sda10f prefix=""/>
+	<!-- xacro below should not be used, use form above-->
+	<!--xacro:sda10f/-->
+	<xacro:motoman_sda10f prefix="my_prefix_"/>
+	
+	
+	<joint name="base_link_joint" type="fixed" >
+    <origin xyz="1.0 0 0"/>
```

Comment:
Yeah. @shaun-edwards: 2-spaces indenting is the default, but seeing as almost all `xacro`s in this package use tabs, it would be better to stay consistent.